ethsigner/signer/hashicorp/src/test/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommandTest.java

```diff
@@ -24,13 +26,15 @@
 
 public class HashicorpSubCommandTest {
 
+  public static final String TLS_KNOWN_SERVER_FILE = "./knownServerFiles.txt";
   private static final String THIS_IS_THE_PATH_TO_THE_FILE =
       Paths.get("/this/is/the/path/to/the/file").toString();
   private static final String HTTP_HOST_COM = "http://host.com";
   private static final String PORT = "23000";
   private static final String PATH_TO_SIGNING_KEY = Paths.get("/path/to/signing/key").toString();
   private static final String FIFTEEN = "15";
   private final ByteArrayOutputStream commandOutput = new ByteArrayOutputStream();
+
   private HashicorpSubCommand hashiConfig;
```

Comment:
I know you didn't add this code, but is not quite named correctly, it's actually the command not the config itself. Perhaps hashicorpSubCommand?

Reply:
done ... and have introduced beforeEach as well to initialize it.